### PR TITLE
Docs: include OpenStack plugin in community supported providers

### LINKED ISSUE #3606: [New Plugin] Openstack
Hello,

I am maintainer of Velero Openstack plugin and this is request for including in the community supported list of plugins. I was present in your one of your reviews sometime in December and we agreed that this plugin can be listed in the community supported ones. 

Recently I released new version that supports block volume snapshots and by this most of the planned plugin features are finished.

The GitHub repository is located here https://github.com/Lirt/velero-plugin-for-openstack/ and I am going to open PR for including it in documentation and link it here.

### PR DESCRIPTION
**Pull request by Lirt**
Add [OpenStack plugin](https://github.com/Lirt/velero-plugin-for-openstack) to community supported ones.

Fixes #3606 

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

Signed-off-by: Ondrej Vasko <ondrej.vaskoo@gmail.com>

### CODE REVIEW

**Comment by Lirt** [comment]
I have one question, can I rename plugin provider name to `community.velero.io/openstack`? Initially I wanted to use `velero.io` but then I found that it's intended for your official plugins. Then I found out I cannot use just `openstack` because the name must have DNS part. Also I don't want to name it according to my username, but rather keep something more community related.

**Comment by nrb** [comment]
> I have one question, can I rename plugin provider name to community.velero.io/openstack? 

I would recommend not using velero.io, unfortunately. We would like to keep that for officially-maintained plugins and resources.

It does have to be a DNS-formatted name but it doesn't have to resolve via a query. Would `velero.openstack.com/cinder` and `velero.openstack.com/swift` work, perhaps?

**Comment by Lirt** [comment]
Hi @nrb 

> It does have to be a DNS-formatted name but it doesn't have to resolve via a query. Would `velero.openstack.com/cinder` and `velero.openstack.com/swift` work, perhaps?

Good idea. I can stick to openstack org `community.openstack.org/cinder` and `community.openstack.org/swift`.

**Comment by Lirt** [comment]
Any update to this PR?

**Review by dsu-igeek — DISMISSED** [review]
These are good, thanks!  We released 1.6, would you add to the v1.6 supported-providers file as well?  Sorry for slipping in another one.

**Comment by Lirt** [comment]
@dsu-igeek Sorry for delay. I merged with latest `main` branch and updated the docs for version 1.6.

File: site/content/docs/main/supported-providers.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Contact: [#Velero Slack](https://kubernetes.slack.com/messages/velero), [GitHub
 | [DigitalOcean](https://www.digitalocean.com/)        | DigitalOcean Object Storage  | DigitalOcean Volumes Block Storage | [StackPointCloud](https://github.com/StackPointCloud/ark-plugin-digitalocean)  |                                 |
 | [Hewlett Packard](https://www.hpe.com/us/en/storage.html)     | 🚫                           | HPE Storage                        | [Hewlett Packard](https://github.com/hpe-storage/velero-plugin)  | [Slack](https://slack.hpedev.io/), [GitHub Issue](https://github.com/hpe-storage/velero-plugin/issues) |
 | [OpenEBS](https://openebs.io/)             | 🚫                           | OpenEBS CStor Volume               | [OpenEBS](https://github.com/openebs/velero-plugin)          | [Slack](https://openebs-community.slack.com/), [GitHub Issue](https://github.com/openebs/velero-plugin/issues) |
+| [OpenStack](https://www.openstack.org/) | Swift | Cinder | [Velero OpenStack Plugin Documentation](https://github.com/Lirt/velero-plugin-for-openstack) | [GitHub Issue](https://github.com/Lirt/velero-plugin-for-openstack/issues) |
```
Comment (jenting) [inline site/content/docs/main/supported-providers.md:32]: nit: I'm thinking if the naming use `OpenStack` would be better?
```suggestion
| [OpenStack](https://www.openstack.org/) | Swift | Cinder | [OpenStack](https://github.com/Lirt/velero-plugin-for-openstack) | [GitHub Issue](https://github.com/Lirt/velero-plugin-for-openstack/issues) |
```

Comment (Lirt) [reply site/content/docs/main/supported-providers.md:32]: Both are OK for me.

Comment (Lirt) [reply site/content/docs/main/supported-providers.md]: Updated in https://github.com/vmware-tanzu/velero/pull/3607/commits/f3f4178f78ad47212e0f1ded4408e2ee04e6cab9

**Review by jenting — APPROVED** [review]

**Review by a-mccarthy — APPROVED** [review]
lgtm!

**Comment by a-mccarthy** [comment]
looks like all requested changes have been addressed cc: @dsu-igeek

**Pull request merged**